Tests/LanguageTranslatorV2Tests/LanguageTranslatorTests.swift

```diff
@@ -136,14 +136,14 @@ class LanguageTranslatorTests: XCTestCase {
         let creationExpectation = self.expectation(description: "Create a custom language model.")
         let deletionExpectation = self.expectation(description: "Delete the custom language model.")
 
-        #if os(iOS)
+        #if os(Linux)
+            let glossary = URL(fileURLWithPath: "Tests/LanguageTranslatorV2Tests/glossary.tmx")
+        #else
             let bundle = Bundle(for: type(of: self))
```

Comment:
Let's make a separate PR for the changes to the platform directives, since those don't require generator changes.

Reply:
This has now been raised as a new PR here: https://github.com/watson-developer-cloud/swift-sdk/pull/816